test/ops/simple.jl

```diff
@@ -1,5 +1,11 @@
+_constrained_input_data = Block([
```

Comment:
maybe `_b_constrained`? Just since elsewhere in the tests we're using the `b_` prefix to indicate something of type `Block`.